README.md

```diff
@@ -26,6 +26,8 @@ The available options are (required options in **bold**, repeatable options in *
 * ***--targetFile***: a source file in which to search for target methods
 * ***--targetMethod***: a target method that must be preserved, and whose dependencies should be stubbed out. Use the format `class.fully.qualified.Name#methodName(Param1Type, Param2Type, ...)`
 * **--outputDirectory**: the directory in which to place the output. The directory must be writeable and will be created if it does not exist.
+* *--jarPath*: the path of a Jar file for Specimin to take as input. 
```

Comment:
This should specify whether the path can be relative, absolute, or both.

Reply:
Also, mention that this argument is repeatable (`targetFile` and `targetMethod` should also say that, which you can easily fix here too).

Reply:
Professor, about the repeatable feature, the document you wrote in the past has already specify that. Here is the whole paragraph:

The available options are (required options in **bold**, repeatable options in *italics*):
* **--root**: specifies the root directory of the target project.
* ***--targetFile***: a source file in which to search for target methods
* ***--targetMethod***: a target method that must be preserved, and whose dependencies should be stubbed out. Use the format `class.fully.qualified.Name#methodName(Param1Type, Param2Type, ...)`
* **--outputDirectory**: the directory in which to place the output. The directory must be writeable and will be created if it does not exist.
* *--jarPath*: the path of a Jar file for Specimin to take as input.